src/components/TextInput/BaseTextInput.js

```diff
@@ -219,7 +219,11 @@ class BaseTextInput extends Component {
                 this.props.autoGrowHeight && {scrollPaddingTop: 2 * maxHeight},
             ],
             (finalStyles, s) => ({...finalStyles, ...s}),
-            {},
+            {
+                '--active-label-translate-y': `${styleConst.ACTIVE_LABEL_TRANSLATE_Y}px`,
+                '--active-label-scale': `${styleConst.ACTIVE_LABEL_SCALE}`,
+                '--label-transition-duration': '80ms',
+            },
```

Comment:
Sure!

Reply:
Are you sure? I thought I'll make more sense in `BaseTextInput` as everything is already imported there. This is how the changes will look.

```js
render() {
    const isLabeledMultiline = Boolean(this.props.label.length) && this.props.multiline;
    const labelAnimationStyle = {
        '--active-label-translate-y': `${styleConst.ACTIVE_LABEL_TRANSLATE_Y}px`,
        '--active-label-scale': `${styleConst.ACTIVE_LABEL_SCALE}`,
        '--label-transition-duration': '80ms',
    };

    return (
        <View style={labelAnimationStyle}>
            <BaseTextInput
                // eslint-disable-next-line react/jsx-props-no-spreading
                {...this.props}
                innerRef={(el) => {
                    this.textInput = el;
                    if (!this.props.innerRef) {
                        return;
                    }

                    if (_.isFunction(this.props.innerRef)) {
                        this.props.innerRef(el);
                        return;
                    }

                    this.props.innerRef.current = el;
                }}
                inputStyle={[styles.baseTextInput, styles.textInputDesktop, isLabeledMultiline ? styles.textInputMultiline : {}, ...this.props.inputStyle]}
            />
        </View>
    );
}
```

Reply:
Also, in case you plan to change the animation duration in future, you'll have to do it here too. Unless maybe you want to make that a constant in `stylesConst`?